app/javascript/src/components/editor/Compiler.svelte

```diff
@@ -1,10 +1,18 @@
 <script>
-  import { fly } from "svelte/transition"
   import { compile } from "../../utils/compiler/compile"
+  import { addAlert } from "../../lib/alerts"
   import { copyValueToClipboard } from "../../copy"
 
+  export let inline = false
+
   let compiling = false
-  let copied = false
+  let classes
+
+  $: classes = [
+    "tooltip",
+    $$restProps["class"],
+    !inline && "button button--secondary button--square"
+  ].filter((c) => !!c).join(" ")
```

Comment:
This is kind of hard to read, I feel it would be better to just do it inline on the element itself.